### PR TITLE
add: automated upgrade process RFC

### PR DESCRIPTION
**Pull request by piontec**

### CODE REVIEW

**Comment by pipo02mix** [comment]
yeah @piontec WDYT about integrating with the existing one?

**Comment by piontec** [comment]
Yeah, makes sense, I need to merge them

**Review by kubasobon — APPROVED** [review]

**Review by MarcelMue — APPROVED** [review]

File: automatic-app-upgrades/README.md [hunk 1]
```diff
@@ -55,12 +69,26 @@ More examples are presented in [the documentation](https://fluxcd.io/docs/compon
...
+## Automated upgrades release process summary
+
+The release process with upgrades as described above looks like this:
+
+1. App part
+   1. A team responsible for the app prepares a new app branch for the release
+   1. The new App Chart CR is tested for successful [upgrade using `ats`](https://github.com/giantswarm/app-test-suite#quick-start)
+   1. The App is released, the chart has to be put into OCI repository (already supported by `architect-orb`)
+1. GitOps configuration part
+   1. Customer has prepared GitOps repository, including bases (they don't have to be bases, they can be individual clusters/apps, but we're encouraging bases for better (re)usability) that make sense for grouping their infrastructure (Honeybadger is working on specific layout recommendations)
```
Comment (MarcelMue) [inline automatic-app-upgrades/README.md:81]: I think this structure is really the important thing.

**Comment by gianfranco-l** [comment]
@MarcelMue @piontec what is holding us back from merging [this PR](https://github.com/giantswarm/rfc/pull/36) and close [this ticket](https://github.com/giantswarm/roadmap/issues/1031)?

**Comment by MarcelMue** [comment]
> @MarcelMue @piontec what is holding us back from merging [this PR](https://github.com/giantswarm/rfc/pull/36) and close [this ticket](https://github.com/giantswarm/roadmap/issues/1031)?

IMO nothing is stopping this PR to be merged but it only addresses some points from the issue.

**Pull request merged**